packages/common/src/seda/seda-chain.ts

```diff
@@ -465,6 +469,7 @@ export class SedaChain extends EventEmitter<EventMap> {
 			logger.error(`Could not find callback for message id: ${txMessage.value.id}: ${txMessage.value}`, {
 				id: txMessage.value.traceId,
 			});
+			// TODO: Discuss how do we handle this ERROR for alerting & monitoring.
```

Comment:
HIGH: There is something weird and fishy going on in the node, this is not supposed to happen but it will not interfere with execution.